app/__init__.py

```diff
@@ -28,22 +28,17 @@
     'X-Content-Type-Options': 'nosniff',
 }
 
-LOG_NAME = "eq.log"
-LOG_SIZE = 1048576
-LOG_NUMBER = 10
-
-logger = logging.getLogger(__name__)
-
 cache = Cache()
+logger = get_logger()
 
 
 def rabbitmq_available():
     submitter = SubmitterFactory.get_submitter()
     if submitter.send_test():
-        logging.info('RabbitMQ Healthtest OK')
+        logger.info('rabbitmq healthtest ok')
```

Comment:
It's suggested event messages are lowercase in https://digitaleq.atlassian.net/wiki/pages/viewpage.action?spaceKey=SDC&title=Logging